flake8_comprehensions.py

```diff
@@ -28,6 +28,7 @@ def __init__(self, tree, *args, **kwargs):
         'C406': 'C406 Unnecessary {type} literal - rewrite as a dict literal.',
         'C407': "C407 Unnecessary list comprehension - '{func}' can take a generator.",
         'C408': "C408 Unnecessary {type} call - rewrite as a literal.",
+        'C411': 'C411 Unnecessary list call - rewrite as list comprehension.',
```

Comment:
Perhaps clearer: "remove the outer call to list()" ? No need to "rewrite as a list comprehension" because it already is one...